content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -124,69 +70,41 @@ The master is the machine where the control plane components run, including
 etcd (the cluster database) and the API server (which the kubectl CLI
 communicates with).
 
-To initialize the master, first choose the pod network plugin you want and check if it requires any parameters to be passed to kubeadm while initializing the cluster. Pick one of the machines you previously installed
-kubeadm on, and run:
+1. Choose a pod network add-on and verify if it requires any arguments to 
+be passed to kubeadm initialization. Depending on what
+third-party provider you choose, you might have to set the `--pod-network-cidr` to
+something provider-specific. The tabs below will contain a notice about what flags
+on `kubeadm init` are required.
+1. (Optional) Unless otherwise specified, kubeadm uses the default gateway's network interface
```

Comment:
I think it might be a bit clearer to say: the network interface associated with the default gateway